platform-hub-web/src/app/shared/kube-config-helper-popup.controller.js

```diff
@@ -0,0 +1,22 @@
+export const KubeConfigHelperPopupController = function ($mdDialog, icons, kubeId, token) {
+  'ngInject';
+
+  const ctrl = this;
+
+  ctrl.copyIcon = icons.copy;
+
+  ctrl.clusterName = token.cluster.name;
+
+  const contextName = `${token.cluster.name}_${token.project.shortname}`;
+  const userName = `${kubeId}_${contextName}`;
+
+  ctrl.clusterCommands = `kubectl config set-cluster ${token.cluster.name} --server=${token.cluster.api_url} && kubectl config set clusters.${token.cluster.name}.certificate-authority-data ${token.cluster.ca_cert_encoded}`;
```

Comment:
Do we allow dots in the cluster name definition? If so, it might break the second half of this particular command.

Reply:
Nope. it's only alphanumerics and dashes so should be OK.

Reply:
Looks like [we don't allow dots](https://github.com/UKHomeOffice/platform-hub/blob/8c499f4a80a7af05fe112a847ede4206de3a609d/platform-hub-api/app/models/kubernetes_cluster.rb#L3) so should be all good (but I like your line of thinking!)